rbkit-lib/ui/appmainwindow.cpp

```diff
@@ -8,8 +8,36 @@
 
 AppMainwindow::AppMainwindow(QWidget *parent) : QMainWindow(parent)
 {
-    centralWidget = new CentralWidget(this);
-    setCentralWidget(centralWidget);
+    QWidget *mainWidget = new QWidget(this);
+
+    mainLayout = new QVBoxLayout();
```

Comment:
centralwidget is only deleted once the rbkit-client itself closes
rest its there all the time in the ui
and we have only one instantiation of centralwidget per app instance

this is also true for mainWidget and mainLayout